src/components/rangeslider/draw.js

```diff
@@ -92,8 +95,13 @@ module.exports = function(gd) {
         opts._height = (fullLayout.height - margin.b - margin.t) * opts.thickness;
         opts._offsetShift = Math.floor(opts.borderwidth / 2);
 
-        var x = margin.l + (graphSize.w * domain[0]),
-            y = fullLayout.height - opts._height - margin.b;
+        var x = margin.l + (graphSize.w * domain[0]);
+
+        var y = (
+            margin.t + graphSize.h * (1 - oppDomain[0]) +
+            tickHeight +
+            opts._offsetShift + constants.extraPad
+        );
```

Comment:
@alexcjohnson some improvements on:

 https://github.com/plotly/plotly.js/compare/multiple-rangesliders...rangeslider-crisp-round

I think the mocks look better to my :eyes: , would you mind taking a look at them?